precompiles/referenda/Referenda.sol

```diff
@@ -47,6 +57,38 @@ interface Referenda {
     /// @custom:selector 34038146
     function trackInfo(uint16 trackId) external view returns (TrackInfo memory);
 
+    /// Return the ReferendumStatus for the input referendumIndex
+    /// @param referendumIndex The index of the referendum
+    /// @custom:selector 8d407c0b
+    function referendumStatus(uint32 referendumIndex)
+        external
+        view
+        returns (ReferendumStatus);
+
+    /// Return the referendumInfo for a closed referendum
+    /// @param referendumIndex The index of the referendum
+    /// @custom:selector 078e5678
+    // todo: is this return struct too long, how long is too long, how to test and enforce this
+    function closedReferendumInfo(uint32 referendumIndex)
+        external
+        view
+        returns (
+            ReferendumStatus,
+            uint256,
+            address,
+            uint256,
+            address,
+            uint256
+        );
```

Comment:
cc @nanocryk how do we know when the tuple struct is too long to use as return value?

should we just make return tuples into structs with any size above 2 elements

Reply:
making it into a struct